code/modules/atmospherics/machinery/components/fusion/hfr_procs.dm

```diff
@@ -188,7 +236,12 @@
 	if(machine_stat & (NOPOWER|BROKEN))
 		return FALSE
 	if(use_power == ACTIVE_POWER_USE)
-		update_mode_power_usage(ACTIVE_POWER_USE, (power_level + 1) * MIN_POWER_USAGE) //Max around 350 KW
+		// Up to 350kW from fusion power level
```

Comment:
note for @LemonInTheDark what you think of raising this one? i wouldn't mind making the HFR consume a bit more power than this (remember at the start it used to consume 1.7MW)

Reply:
For what it's worth, peaks of 1250kW - 1350kW are already very hard to sustain, even if you put in local power generation and try to limit use. I'd already be tempted to add a nearby SMES/dedicated APC on maps where the HFR doesn't already have them (Tram does) so people can keep a closer eye on what's going on

Reply:
I don't mind all that much, so long as it doesn't hinder setup time to a hellish degree. We want people using this thing after all